nucleus/osgi-platforms/felix/src/main/resources/config/osgi.properties

```
@@ -580,32 +580,77 @@ org.osgi.framework.system.capabilities= \
  ${eecap-${java.specification.version}}
```

Thinking about it once again, I believe we need only this:

```suggestion
 osgi.ee;osgi.ee="JavaSE/compact1";version="1.8", \
 osgi.ee;osgi.ee="JavaSE/compact2";version="1.8",\
 osgi.ee;osgi.ee="JavaSE/compact3";version="1.8", ${eecap-${java.specification.version}}
```
Insteead of all changes below. Compact versions only existed in Java 1.8 and were superseded with Jigsaw. This should be enough to make osgi.function happy.

Let me quickly try that out...

---

Yep that works well.